.github/workflows/config.yml

```diff
@@ -89,7 +89,7 @@ jobs:
 
       - name: Run buildstockbatch
         run: |
-          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          pip install git+https://github.com/NREL/buildstockbatch.git@workflow-refactor
```

Comment:
Does this automatically get reverted to develop upon merge?

Reply:
No. I'll manually revert it before merge.